modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -32,28 +32,28 @@ ip-10-0-170-165.ec2.internal audit.log
 .. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
```

Comment:
I'm confused what you mean by `log for a specific ... timestamp or view all the logs`? Do you mean there is a command `for a specific ... timestamp`, and there is a command `view all the logs`? If yes, could you give the command example respectively?